fastn-core/src/library2022/processor/sqlite.rs

```diff
@@ -190,6 +169,30 @@ fn resolve_variable_from_headers(
     Ok(param_value)
 }
 
+fn resolve_param(
+    param_name: &str,
+    param_type: &str,
+    doc: &ftd::interpreter::TDoc,
+    headers: &ftd::ast::HeaderValues,
+    line_number: usize,
+) -> ftd::interpreter::Result<Box<dyn rusqlite::ToSql>> {
+    resolve_variable_from_headers(param_name, param_type, doc, headers, line_number)
+        .or_else(|_| resolve_variable_from_doc(param_name, doc, line_number))
+        .map(|v| Box::new(v) as Box<dyn rusqlite::ToSql>)
+}
+
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, PartialEq)]
```
Use `PartialEq`to use `.eq()`